IntegrationTests/allocation-counter-tests-framework/template/AtomicCounter/Sources/AtomicCounter/include/atomic-counter.h

```diff
@@ -22,4 +22,8 @@ void inc_malloc_counter(void);
 void reset_malloc_counter(void);
 long read_malloc_counter(void);
 
+void add_malloc_bytes_counter(int v);
```

Comment:
This function doesn't exist: `MAKE_COUNTER` doesn't define an `add_counter` function and you haven't extended the macro to use this, so I suspect this would fail to link.

Reply:
Right, sorry; sloppy copy-pasta-ing; In general, we happy to merge such new counter?

Reply:
@weissi owns this mostly. I don't care, but he may care.

Reply:
I'm happy with that. But the type should match `malloc`'s argument and be `size_t`.

Reply:
Right, thanks :) Sorry for the messy PR, should have vetted my hacky thing before I cherry picked it over